app/form_models/jobseekers/job_application/personal_details_form.rb

```diff
@@ -1,7 +1,8 @@
 class Jobseekers::JobApplication::PersonalDetailsForm
   include ActiveModel::Model
 
-  attr_accessor :first_name
+  attr_accessor :building_and_street, :county, :email_address, :first_name, :last_name, :national_insurance_number,
+  :phone_number, :previous_names, :postcode, :teacher_reference_number, :town_or_city
 
-  validates :first_name, presence: true
+  include JobApplicationPersonalDetailsValidations
```

Comment:
Why are you doing this? is this module required elsewhere?

Reply:
No this should've been removed!

Reply:
It's been removed since it seems.